### ISSUE TITLE
CoolLED out of date

### ISSUE THREAD

**Issue by dstoychev** [python-microscope/microscope#190]
The CoolLED implementation seems out of date. It doesn't use the new trigger mixins, and it implements enable/disable methods directly instead of using _do_enable and _do_disable. There may be other issues as well. I think it needs to be reviewed and brought up-to-date.

**Comment by carandraug** [comment]
You're right that it implements `enable` and `disable` instead of `_do_enable` and `_do_disable`, but it does implement the `TriggerTargetMixin`. Are you sure you're looking at the latest master branch? This was done in 3d69829abc919
